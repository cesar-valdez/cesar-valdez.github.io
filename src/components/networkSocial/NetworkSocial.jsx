import React from 'react';
import ReactDOM from 'react-dom';

import './networkSocial.scss'

const Networks = () => {
    return (
			<div className="pi_Card-network">
				<div className="pi_Card-network-location">
						<i data-icon="ei-location" data-size="s"></i>
						<span>Saltillo, Coahuila</span>
				</div>

				<ul className="pi_Card-network-contact">
						<li className="pi_Card-network-contact-item">
								<a href="" target="_blank">
										<i data-icon="ei-envelope" data-size="m"></i>
								</a>
						</li>
						<li className="pi_Card-network-contact-item">
								<a href="https://github.com/cesar-valdez" target="_blank">
										<i data-icon="ei-sc-github" data-size="m"></i>
								</a>
						</li>
						<li className="pi_Card-network-contact-item">
								<a href="https://www.linkedin.com/in/c%C3%A9sar-alberto-bustos-valdez-7b6a8a154/" target="_blank">
										<i data-icon="ei-sc-linkedin" data-size="m"></i>
								</a>
						</li>
				</ul>
		</div>
    )
}

export default Networks;