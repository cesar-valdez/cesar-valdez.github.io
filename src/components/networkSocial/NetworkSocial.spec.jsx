
import React from 'react';
import {shallow} from 'enzyme';

import NetworkSocial from './NetworkSocial';

describe('NetworkSocial', () => {
	const networkSocial = shallow(<NetworkSocial />);
	
	it('should render Network social component', () => {
		expect(networkSocial).toMatchSnapshot();
	});
});