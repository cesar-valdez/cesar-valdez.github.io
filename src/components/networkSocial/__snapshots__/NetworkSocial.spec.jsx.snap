// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NetworkSocial should render Network social component 1`] = `
<div
  className="pi_Card-network"
>
  <div
    className="pi_Card-network-location"
  >
    <i
      data-icon="ei-location"
      data-size="s"
    />
    <span>
      Saltillo, Coahuila
    </span>
  </div>
  <ul
    className="pi_Card-network-contact"
  >
    <li
      className="pi_Card-network-contact-item"
    >
      <a
        href=""
        target="_blank"
      >
        <i
          data-icon="ei-envelope"
          data-size="m"
        />
      </a>
    </li>
    <li
      className="pi_Card-network-contact-item"
    >
      <a
        href="https://github.com/cesar-valdez"
        target="_blank"
      >
        <i
          data-icon="ei-sc-github"
          data-size="m"
        />
      </a>
    </li>
    <li
      className="pi_Card-network-contact-item"
    >
      <a
        href="https://www.linkedin.com/in/c%C3%A9sar-alberto-bustos-valdez-7b6a8a154/"
        target="_blank"
      >
        <i
          data-icon="ei-sc-linkedin"
          data-size="m"
        />
      </a>
    </li>
  </ul>
</div>
`;
